IDL/structures.idl

```diff
@@ -0,0 +1,91 @@
+enum MyEnum
+{
+    ENUM_VALUE_1,
+    ENUM_VALUE_2,
+    ENUM_VALUE_3
+};
+
+@bit_bound(8)
+bitmask MyBitMask
+{
+    @position(0) flag0,
+    @position(1) flag1,
+    @position(4) flag4,
+    @position(6) flag6
+};
+
+typedef long MyAlias;
+
+struct MyStructure
+{
+    long field1;
+    float field2;
+};
+
+union MyUnion switch (long)
+{
+    case 0: long longValue;
+    case 1: float floatValue;
+};
+
+bitset MyBitset
+{
+    bitfield<3> a;
+    bitfield<1> b;
+    bitfield<4>;
+    bitfield<10> c;
+    bitfield<12, short> d;
+};
+
+struct Structures
+{
+    short var_short;
+    unsigned short var_ushort;
+    long var_long;
+    unsigned long var_ulong;
+    long long var_longlong;
+    unsigned long long var_ulonglong;
+    float var_float;
+    double var_double;
+    long double var_longdouble;
+    boolean var_boolean;
+    octet var_octet;
+    char var_char8;
+    wchar var_char16;
+    string var_string8;
+    wstring var_string16;
+
+    MyEnum var_enum;
+    MyBitMask var_bitmask;
+    MyAlias var_alias;
+    short var_array_short[10];
+    sequence<long> var_sequence;
+    map<long, long> var_map;
+    MyUnion var_union;
+    MyStructure var_structure;
+    MyBitset var_bitset;
+};
+
+struct MyStructureChild : MyStructure
+{
+    long field3;
+    float field4;
+};
+
+struct MyStructureEmptyChild: MyStructure
+{
+};
+
+struct MyEmptyStructure
+{
+};
+
+struct MyEmptyStructureChild : MyEmptyStructure

```

Comment:
The structures in `includes.idl` and these ones with inherintance, I would move it to a idl file `inheritance.idl`.

Reply:
moved in https://github.com/eProsima/dds-types-test/pull/1/commits/7807affbc3bdb9302cd109f1e64c8a0bf837d7c8